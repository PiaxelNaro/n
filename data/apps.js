export default [
    {
        id: 1,
        icon:
            'https://static-s.aa-cdn.net/img/gp/20600003867506/WjPDaHXE8ufMDQ43x8qpLfH9yuw1D0mlKzOJLSbeETcRLCONpBChskUtj3DtVTDJknw=w300_w80',
        company: 'Networkcanal Multimedia Ltd',
        name: 'Fa',
    },
    {
        id: 2,
        icon:
            'https://static-s.aa-cdn.net/img/gp/20600004729627/fFI2bIE44LYBwecU17bVeIdvKbRao6O3zek6U6x-5A22uXbEO4fWm15dfffg2jHBTg=w300_w80',
        company: 'KAPS Development',
        name: 'FA',
    },
    {
        id: 3,
        icon:
            'https://static-s.aa-cdn.net/img/ios/1039182706/18de8684db7bcaebb95232d7dc66f252_w80',
        company: 'INNERGY CO.,LTD',
        name: '와이드앵글 FA 교육',
    },
    {
        id: 4,
        icon:
            'https://static-s.aa-cdn.net/img/ios/358534918/ec96f2e8d1fdb23f5cea8671d9d8335a_w80',
        company: '7700 Software',
        name: 'FA World',
    },
    {
        id: 5,
        icon:
            'https://static-s.aa-cdn.net/img/ios/688824991/b3ca3dbc0fb23363af7aa7917d4728a8_w80',
        company: 'Lubomir Havran BINARTS',
        name: 'FA Magazín',
    },
    {
        id: 6,
        icon:
            'https://static-s.aa-cdn.net/img/ios/805530850/d5b350ceb68604faafe6a3c234b07bb7_w80',
        company: 'Tobit',
        name: 'FA/KE',
    },
    {
        id: 7,
        icon:
            'https://static-s.aa-cdn.net/img/ios/884096407/1f6d3ba08a891906570ad2bb68b8dcdd_w80',
        company: 'Tobit',
        name: 'Fa. Handelsagent',
    },
    {
        id: 8,
        icon:
            'https://static-s.aa-cdn.net/img/ios/892841769/b27b0d7ccf10c6182fcf6ddf30017421_w80',
        company: 'Spinn Labs Private Limited',
        name: 'Bao Fa',
    },
    {
        id: 9,
        icon:
            'https://static-s.aa-cdn.net/img/ios/906627580/f41757b0e6c2509e20a937e80e2e0233_w80',
        company: 'E Sports Publications Limited',
        name: 'Lincolnshire FA',
    },
    {
        id: 10,
        icon:
            'https://static-s.aa-cdn.net/img/ios/919412577/ad1ae851ddf94c080bc93ad41c0d20fe_w80',
        company: 'Tobit',
        name: 'FA Foto',
    },
    {
        id: 11,
        icon:
            'https://static-s.aa-cdn.net/img/gp/20600003867506/WjPDaHXE8ufMDQ43x8qpLfH9yuw1D0mlKzOJLSbeETcRLCONpBChskUtj3DtVTDJknw=w300_w80',
        company: 'Networkcanal Multimedia Ltd',
        name: 'Fa1',
    },
    {
        id: 12,
        icon:
            'https://static-s.aa-cdn.net/img/gp/20600004729627/fFI2bIE44LYBwecU17bVeIdvKbRao6O3zek6U6x-5A22uXbEO4fWm15dfffg2jHBTg=w300_w80',
        company: 'KAPS Development',
        name: 'FA1',
    },
    {
        id: 13,
        icon:
            'https://static-s.aa-cdn.net/img/ios/1039182706/18de8684db7bcaebb95232d7dc66f252_w80',
        company: 'INNERGY CO.,LTD',
        name: '와이드앵글 FA 교육1',
    },
    {
        id: 14,
        icon:
            'https://static-s.aa-cdn.net/img/ios/358534918/ec96f2e8d1fdb23f5cea8671d9d8335a_w80',
        company: '7700 Software',
        name: 'FA World1',
    },
];
