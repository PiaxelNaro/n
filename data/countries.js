const countries = {
    ALL: {
        code: 'ALL',
        name: 'All Countries',
    },
    AU: {
        code: 'AU',
        name: 'Australia',
        isLocked: true,
    },
    US: {
        code: 'US',
        name: 'America',
    },
    KO: {
        code: 'KO',
        name: 'South of Korea',
        isLocked: true,
    },
    LG: {
        code: 'LG',
        name: 'Long Long Name Country',
    },
    UK: {
        code: 'UK',
        name: 'England',
    },
};

const categories = [
    {
        name: 'All Countries',
        countries: ['ALL', 'AU', 'US', 'KO', 'LG', 'UK'],
    },
    {
        name: 'Most Popular',
        countries: ['ALL', 'AU', 'US', 'UK'],
    },
];

export { categories, countries };
