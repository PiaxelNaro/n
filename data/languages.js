export default [
    {
        code: 'en-US',
        label: 'English',
    },
    {
        code: 'ja-JP',
        label: '日本語',
    },
    {
        code: 'ko-KR',
        label: '한국어',
    },
    {
        code: 'zh-CN',
        label: '中文(简体)',
    },
    {
        code: 'ru-RU',
        label: 'Русский',
    },
    {
        code: 'fr-FR',
        label: 'Français',
    },
    {
        code: 'de-DE',
        label: 'Deutsch',
    },
];
