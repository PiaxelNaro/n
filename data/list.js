export default {
    current: 'games',
    list: [
        {
            title: 'Overall',
            id: 'overall',
        },
        {
            title: 'Games',
            id: 'games',
            subList: [
                {
                    title: 'Action',
                    id: 'action',
                },
                {
                    title: 'Adventure',
                    id: 'adventure',
                },
                {
                    title: 'Arcade',
                    id: 'arcade',
                },
                {
                    title: 'Board',
                    id: 'board',
                },
                {
                    title: 'Card',
                    id: 'card',
                },
                {
                    title: 'Casino',
                    id: 'casino',
                },
                {
                    title: 'Dice',
                    id: 'dice',
                },
                {
                    title: 'Education',
                    id: 'game_education',
                },
                {
                    title: 'Family',
                    id: 'family',
                },
                {
                    title: 'Kids',
                    id: 'game_kids',
                },
                {
                    title: 'Music',
                    id: 'game_music',
                },
                {
                    title: 'Puzzle',
                    id: 'puzzle',
                },
                {
                    title: 'Racing',
                    id: 'racing',
                },
                {
                    title: 'Role Playing',
                    id: 'role_playing',
                },
                {
                    title: 'Simulation',
                    id: 'simulation',
                },
                {
                    title: 'Sports',
                    id: 'sports',
                },
                {
                    title: 'Strategy',
                    id: 'strategy',
                },
                {
                    title: 'Trivia',
                    id: 'trivia',
                },
                {
                    title: 'Word',
                    id: 'word',
                },
            ],
        },
        {
            title: 'Applications',
            id: 'applications',
        },
        {
            title: 'Books',
            id: 'books',
        },
        {
            title: 'Business',
            id: 'business',
        },
        {
            title: 'Catalogs',
            id: 'catalogs',
        },
        {
            title: 'Education',
            id: 'education',
        },
        {
            title: 'Entertainment',
            id: 'entertainment',
        },
        {
            title: 'Finance',
            id: 'finance',
        },
        {
            title: 'Food and Drink',
            id: 'food_and_drink',
        },
        {
            title: 'Health and Fitness',
            id: 'health_and_fitness',
        },
        {
            title: 'Kids',
            id: 'kids',
            subList: [
                {
                    title: 'Kids 5 & Under',
                    id: 'kids_5_under',
                },
                {
                    title: 'Kids Age 6-8',
                    id: 'kids_age_6_8',
                },
                {
                    title: 'Kids Age 9-11',
                    id: 'kids_age_9_11',
                },
            ],
        },
        {
            title: 'Lifestyle',
            id: 'lifestyle',
        },
        {
            title: 'Magazines and Newspapers',
            id: 'magazines_and_newspapers',
        },
        {
            title: 'Medical',
            id: 'medical',
        },
        {
            title: 'Music',
            id: 'music',
        },
        {
            title: 'Navigation',
            id: 'navigation',
        },
        {
            title: 'News',
            id: 'news',
        },
        {
            title: 'Photo and Video',
            id: 'photo_and_video',
        },
        {
            title: 'Productivity',
            id: 'productivity',
        },
        {
            title: 'Reference',
            id: 'reference',
        },
        {
            title: 'Shopping',
            id: 'shopping',
        },
        {
            title: 'Social Networking',
            id: 'social_networking',
        },
        {
            title: 'Sports',
            id: 'sports',
        },
        {
            title: 'Travel',
            id: 'travel',
        },
        {
            title: 'Utilities',
            id: 'utilities',
        },
        {
            title: 'Weather',
            id: 'weather',
        },
    ],
};
