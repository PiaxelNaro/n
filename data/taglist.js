export default [
    {
        id: 'ad_revenue',
        name: 'Ad Revenue',
    },
    {
        id: 'ecpm',
        name: 'eCPM',
    },
    {
        id: 'ecpc',
        name: 'eCPC',
    },
    {
        id: 'click',
        name: 'Clicks',
    },
    {
        id: 'fill_rate',
        name: 'Fill Rate',
    },
];
