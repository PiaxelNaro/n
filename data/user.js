export default {
    profile: {
        surveyFirstName: '',
        surveyName: '',
        email: '',
        fileRepository: 'repository_788',
    },
    isANA: true,
    isINT: true,
    isApiPartner: true,
    intAuth: {
        store: {
            pp: true,
            pya: true,
            pyp: false,
        },
        usage: {
            pp: true,
            pya: true,
        },
        marketing: {
            advertising: false,
            aso: false,
        },
    },
};
