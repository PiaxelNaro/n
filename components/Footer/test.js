// @flow
import React from 'react';
import { shallow } from 'enzyme';

import mockI18n from '../../modules/i18n/mockI18n';
import { _Footer as Footer } from '.';

import data from '../../data/footer';

describe('<Footer>', () => {
    it('renders footer', () => {
        const wrapper = shallow(<Footer i18n={mockI18n} content={data} />);
        expect(wrapper).toMatchSnapshot();
    });
});
