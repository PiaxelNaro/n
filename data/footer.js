import languages from './languages';

export default {
    groups: [
        {
            links: [
                {
                    label: '主页',
                    url: '/',
                    newwin: false,
                },
                {
                    label: '销售',
                    url: '/sales',
                    newwin: true,
                }
            ],
        },
        {
            links: [
                {
                    label: '咨询',
                    url: '/consult',
                    newwin: true,
                },
                {
                    label: '关于',
                    url:
                        '/about',
                    newwin: false,
                }
            ],
        },
    ],
    languages_for_switch: languages
};
