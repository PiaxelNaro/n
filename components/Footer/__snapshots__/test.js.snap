// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> renders footer 1`] = `
<Footer__FooterContainer>
  <Footer__FooterMain>
    <ul>
      <li>
        <Footer__TextLink
          href="/?_ref=footer"
        >
          Home
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/tours/enterprise-market-data/"
        >
          Market Data Intelligence
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/tours/app-analytics-platform/"
        >
          App Analytics
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/tours/market-data/"
        >
          Store Stats
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/insights/"
        >
          Insights
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/pricing/"
        >
          Pricing
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/about/"
        >
          About
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/customers/"
        >
          Customers
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/about/leadership/"
        >
          Leadership
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/about/careers/"
        >
          Careers
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/about/partnerships/"
        >
          Partnerships
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/about/press/"
        >
          Press
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/about/contact/"
        >
          Contact
        </Footer__TextLink>
      </li>
    </ul>
    <ul>
      <li>
        <Footer__TextLink
          href="http://support..com/?_ref=footer"
        >
          Support
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="https://support..com/hc/sections/204363128-Market-Data-Intelligence-API"
        >
          Market Data Intelligence API
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/legal/security/"
        >
          Security
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/legal/data-usage-policy/"
        >
          Data Usage Policy
        </Footer__TextLink>
      </li>
      <li>
        <Footer__TextLink
          href="/legal/copyright/"
        >
          Copyright Policy
        </Footer__TextLink>
      </li>
    </ul>
    <Footer__LanguageSelectorWrapper>
      <LanguageSelector
        current="en-US"
        languages={
          Array [
            Object {
              "code": "en-US",
              "label": "English",
            },
            Object {
              "code": "ja-JP",
              "label": "日本語",
            },
            Object {
              "code": "ko-KR",
              "label": "한국어",
            },
            Object {
              "code": "zh-CN",
              "label": "中文(简体)",
            },
            Object {
              "code": "ru-RU",
              "label": "Русский",
            },
            Object {
              "code": "fr-FR",
              "label": "Français",
            },
            Object {
              "code": "de-DE",
              "label": "Deutsch",
            },
          ]
        }
      />
    </Footer__LanguageSelectorWrapper>
  </Footer__FooterMain>
  <Footer__FooterSub>
    <FlexGrid>
      <FlexGrid.Item
        flex={1}
      >
        <ul
          className="footer-legal"
        >
          <li>
            <Footer__CopyRight>
              © 
              2017
               App Annie
            </Footer__CopyRight>
          </li>
          <li>
            <Footer__TextLink
              href="/legal/terms/"
            >
              Terms of Service
            </Footer__TextLink>
          </li>
          <li>
            <Footer__TextLink
              href="/legal/privacy/"
            >
              Privacy Policy
            </Footer__TextLink>
          </li>
          <li>
            <a
              href="//privacy-policy.truste.com/privacy-seal/seal?rid=fc9050fc-4b23-4d84-87bf-a903d26b16bc"
              rel="noopener noreferrer"
              target="_blank"
              title="TRUSTe"
            >
              <Footer__Img
                alt="TRUSTe"
                height="26"
                src="//privacy-policy.truste.com/privacy-seal/seal?rid=fc9050fc-4b23-4d84-87bf-a903d26b16bc"
              />
            </a>
          </li>
        </ul>
      </FlexGrid.Item>
      <Footer__Social>
        <a
          href="http://www.linkedin.com/company/app-annie"
          rel="noopener noreferrer"
          target="_blank"
        >
          <Footer__SocialIcon
            className="social-icon-linkedin"
          />
        </a>
        <a
          href="http://www.facebook.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <Footer__SocialIcon
            className="social-icon-facebook"
          />
        </a>
        <a
          href="http://www.twitter.com//"
          rel="noopener noreferrer"
          target="_blank"
        >
          <Footer__SocialIcon
            className="social-icon-twitter"
          />
        </a>
        <a
          href="http://google.com/+"
          rel="noopener noreferrer"
          target="_blank"
        >
          <Footer__SocialIcon
            className="social-icon-google_plus"
          />
        </a>
        <a
          href="http://www.slideshare.net/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <Footer__SocialIcon
            className="social-icon-slide_share"
          />
        </a>
        <a
          href="http://www.youtube.com/tv"
          rel="noopener noreferrer"
          target="_blank"
        >
          <Footer__SocialIcon
            className="social-icon-youtube"
          />
        </a>
        <a
          href="http://e.weibo.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <Footer__SocialIcon
            className="social-icon-weibo"
          />
        </a>
        <a
          href="http://blog..com/feed/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <Footer__SocialIcon
            className="social-icon-rss"
          />
        </a>
      </Footer__Social>
    </FlexGrid>
  </Footer__FooterSub>
</Footer__FooterContainer>
`;
